cmd/flags/driver.go

```diff
@@ -43,6 +43,14 @@ var (
 		Usage:    "HTTP RPC endpoint of another synced L2 execution engine node",
 		Category: driverCategory,
 	}
+	// syncer specific flag
+	MaxExponent = &cli.Uint64Flag{
```

Comment:
Not sure if this flag is necessary, we can always simply roll-back to the latest matched verified block hash?

Reply:
And if u want to keep this flag, u also need to add this flag to the `DriverFlags` below.

Reply:
> And if u want to keep this flag, u also need to add this flag to the `DriverFlags` below.

OK

Reply:
> Not sure if this flag is necessary, we can always simply roll-back to the latest matched verified block hash?

If we sync the block from L2 to L3, this value will be larger than when we sync from L1 to L2, so I think it's better to keep it a configurable value